package/moj/components/add-another/add-another.js

```diff
@@ -42,9 +42,13 @@ MOJFrontend.AddAnother.prototype.getNewItem = function() {
 
 MOJFrontend.AddAnother.prototype.updateAttributes = function(index, item) {
 	item.find('[data-name]').each(function(i, el) {
+    var originalId = el.id
+
 		el.name = $(el).attr('data-name').replace(/%index%/, index);
 		el.id = $(el).attr('data-id').replace(/%index%/, index);
-		($(el).siblings('label')[0] || $(el).parents('label')[0]).htmlFor = el.id;
+
+    var label = $(el).siblings('label')[0] || $(el).parents('label')[0] || item.find('[for="' + originalId + '"]')[0];
```

Comment:
item.find isn't compatible with IE11, is this covered by polyfills elsewhere? https://caniuse.com/?search=find

Reply:
Ah, it's not clear from the variable naming but `item` is a jQuery object, so it's [a different .find](https://api.jquery.com/find/)